### PR TITLE
Support release and debug build on ARTIK053

### PR DESCRIPTION
**Pull request by robertsipka**

### CODE REVIEW

File: API/resources/resources.json [hunk 1]
```diff
@@ -37,10 +37,15 @@
       },
       "config": [
         {
-          "condition": "'%{app}' == 'iotjs'",
+          "condition": "'%{app}' == 'iotjs' and '%{build-type}' == 'release'",
           "src": "%{config}/tizenrt-iotjs.config",
```
Comment (rtakacs) [inline API/resources/resources.json]: Please rename the `tizenrt-iotjs.config` file to `tizenrt-iotjs-release.config`.

File: API/resources/resources.json [hunk 2]
```diff
@@ -37,10 +37,15 @@
...
           "dst": "%{tizenrt}/build/configs/artik053/iotjs/defconfig"
         },
+        {
+          "condition": "'%{app}' == 'iotjs' and '%{build-type}' == 'debug'",
+          "src": "%{config}/tizenrt-iotjs-debug.config",
+          "dst": "%{tizenrt}/build/configs/artik053/iotjs/defconfig"
+        },
```
Comment (rtakacs) [inline API/resources/resources.json]: What about if we merge the **release** and **debug** conditions?

```
{
  "condition": "'%{app}' == 'iotjs'"
  "src": "%{config}/tizenrt-iotjs-%{build-type}.config",
  "dst": "%{tizenrt}/build/configs/artik053/iotjs/defconfig"
}
```

File: API/resources/resources.json [hunk 3]
```diff
@@ -52,10 +57,14 @@
           "dst": "%{tizenrt}/build/configs/artik053/jerryscript/"
         },
         {
-          "condition": "'%{app}' == 'jerryscript'",
+          "condition": "'%{app}' == 'jerryscript' and '%{build-type}' == 'release'",
           "src": "%{config}/tizenrt-jerryscript.config",
```
Comment (rtakacs) [inline API/resources/resources.json]: Ditto. Please rename the `tizenrt-jerryscript.config` to `tizenrt-jerryscript-release.config`.

File: API/resources/resources.json [hunk 4]
```diff
@@ -52,10 +57,14 @@
...
           "dst": "%{tizenrt}/build/configs/artik053/jerryscript/defconfig"
-
+        },
+        {
+          "condition": "'%{app}' == 'jerryscript' and '%{build-type}' == 'debug'",
+          "src": "%{config}/tizenrt-jerryscript-debug.config",
+          "dst": "%{tizenrt}/build/configs/artik053/jerryscript/defconfig"
```
Comment (rtakacs) [inline API/resources/resources.json]: Same as above.

Comment (robertsipka) [reply API/resources/resources.json]: @rtakacs : Thanks, it was a good idea. I've removed the condition also (It was needless after replacing the application name and build-type).

**Comment by yichoi** [comment]
LGTM

**Pull request merged**